src/controllers/anniversary_message.py

```diff
@@ -0,0 +1,43 @@
+from typing import List, Tuple
+
+from src.controllers.base import BaseController
+from src.config import EnvManager
+from src.utils import dateutils
+
+
+class AnniversaryMessageController(BaseController):
+
+    gif_keywords: frozenset = frozenset(('work', 'party'))
+
+    @staticmethod
+    def fill_from_template(username: str, anniversary_years: str, template: str) -> str:
+        return template.format(username=username, anniversary_years=anniversary_years)
+
+    @staticmethod
+    def get_anniversary_employees(employees: List[dict], anniversary_field) -> List[dict]:
+        def is_anniversary(anniversary: str):
+            if not anniversary:
+                return False
+            delimiter = '-'
+            current_day_month: tuple = dateutils.get_current_day_month(EnvManager.UTC_HOUR_OFFSET)
+            anniversary_day, anniversary_month, _ = tuple(int(value) for value in reversed(anniversary.split(delimiter)))
```

Comment:
Maybe we can extract this logic to the controller?

Seems very similar to the birthday message selector

Reply:
The function was improved and it makes more sense to be a part of dateutils